notebooks/BTA_array_producer.ipynb

```diff
@@ -0,0 +1,2322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        "../testfiles/nano_mc2018_ULv2.root",
</pre><p>this testfile is not included in the repository, please consider using one from e.g. <code>metadata/test_bta_run3.json</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cms-btv-pog/BTVNanoCommissioning/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Maybe just to clarify here: Is this notebook meant to just illustrate the differences between old BTA and the new BTA-like ntuples? If so, I guess using a Run 3 sample makes no sense, as the old BTA framework does not run with CMSSW 13 (or at least has never been checked with that version), and is likely also not set up to use Puppi jets etc. 

Do we want to have the used MC 2018 UL test file still to be included in this repo such that interested people could re-run the notebook? What do you think @Ming-Yan and @colizz ?

Reply:
My understanding is @colizz use UL18 to test all configuration with coffea based stuff, and make cross-check there. I just propagate this to the commissioning framework(but I leave the run2 config since we don't need in the future).
I am fine with keep run2 file for people to rerun the workbook, we can put the testfiles in the eos btv space and change the corresponding path

Reply:
That's how I understood it as well, and I think putting the test file on eos doesn't harm :)